packages/plugins/plugin-inbox/src/components/Message/Message.tsx

```diff
@@ -61,10 +62,10 @@ export const Message = forwardRef<HTMLDivElement, MessageProps>(
     const { parentRef } = useTextEditor({ initialValue: content, extensions }, [content, extensions]);
 
     return (
-      <div role='none' className='grid grid-rows-[min-content_1fr] relative h-full overflow-hidden'>
+      <div role='none' className='grid grid-rows-[min-content_1fr]'>
         <MessageHeader ref={ref} message={message} viewMode={viewMode} onSenderClick={onSenderClick} />
-        <div role='none' className='overflow-y-auto h-full min-h-0 p-2'>
-          <div ref={parentRef} className={mx(classNames)} />
+        <div role='none' className='relative'>
+          <div role='none' ref={parentRef} className={mx('absolute inset-0', classNames)} />
         </div>
       </div>
```

Comment:
The secret 🛸. ✨

Reply:
I hate doing this, but I could find no other way to get grid to ignore CM’s intrinsic size…